vars/neoDeploy.groovy

```diff
@@ -59,16 +59,31 @@ import static com.sap.piper.Prerequisites.checkScript
      * @mandatory for deployMode=warParams
      */
     'runtimeVersion',
-        /**
+    /**
      * Compute unit (VM) size. Acceptable values: lite, pro, prem, prem-plus.
      * @parentConfigKey neo
      */
     'size',
-        /**
```

Comment:
[nit-pick] unrelated

Reply:
I would like to keep this. Can't we go by the Scouts Rule: _Leave the campground cleaner than you found it.5_?